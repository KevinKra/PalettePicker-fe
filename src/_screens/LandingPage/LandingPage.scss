@import "../../_styles/main.scss";

.LandingPage {
	border: 1px solid coral;
	background-color: $primary-dark;
}
