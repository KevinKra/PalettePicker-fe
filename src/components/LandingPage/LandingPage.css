.LandingPage {
  /* min-height: 100vh; */
  border: 1px solid coral;
  background-color: rgb(40, 48, 56);
}
