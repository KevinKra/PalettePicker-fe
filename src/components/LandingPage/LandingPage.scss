@import "../../_styles/variables.scss";

.LandingPage {
  border: 1px solid coral;
  background-color: $primary-dark;
}
