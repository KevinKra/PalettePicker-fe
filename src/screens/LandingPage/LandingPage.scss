.LandingPage {
	display: grid;
	grid-template-rows: 2fr 1fr;
	min-height: 92vh;
	background-color: rgb(40, 48, 56);
}
