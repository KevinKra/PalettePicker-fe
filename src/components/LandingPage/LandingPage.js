import React from "react";
import ProjectsContainer from "../../containers/ProjectsContainer/ProjectsContainer";
import PalettePicker from "../../containers/PalettePicker/PalettePicker";
import "./LandingPage.css";

export default function LandingPage() {
  return (
    <main className="LandingPage">
      <PalettePicker />
      <ProjectsContainer />
    </main>
  );
}
