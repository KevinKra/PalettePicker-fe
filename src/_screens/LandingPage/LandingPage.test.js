import React from "react";
import LandingPage from "./LandingPage";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

describe("LandingPage", () => {
  it("should match the snapshot", () => {
    const wrapper = shallow(<LandingPage />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
