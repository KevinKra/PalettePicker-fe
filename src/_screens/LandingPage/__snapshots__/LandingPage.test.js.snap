// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should match the snapshot 1`] = `
<main
  className="LandingPage"
>
  <Connect(withRouter(PalettePicker)) />
  <Connect(ProjectsContainer) />
</main>
`;
